front_end/html/edit_exercise.html

```diff
@@ -261,6 +262,18 @@ <h2>Create exercise</h2>
             </div>
         </div>
 
+
+        <div class="shadow-box">
+            <div>
+                <p><strong>checking code : </strong></p>
+                <p>checking code can be used to check certain aspects of student's code (i.e. ensuring that they don't use certain functions.) The student's code is saved as "code" and can be opened to run tests on. If an error is found, print the error to standard output. Otherwise, do nothing.</p>
```

Comment:
Please make sure to capitalize consistently and use consistent spacing throughout the app. This helps ensure that the app looks consistent and professional.

Please change "checking code :" to "Pre-execution logic:".

Please change the description to: "In some cases, instructors wish to check certain aspects of a student's code before it is executed. For example, they may wish to verify that the student has not used certain functions, variable names, etc. If you wish to use this functionality, enter code in the box below. The code must be in the same programming language as the student's. The student's code will be saved in a file called "code" in the current working directory. Your code must read this file and then check the student's code. If a problem is found, print a message to standard output. Otherwise, print nothing to standard output."